README.md

```diff
@@ -265,7 +266,7 @@ docker run -p 8080:80 \
    --num-shard 8
 ```
 
-### LLava-next 7B BF16 on 1 Gaudi2 card
+### llava-v1.6-mistral-7b-hf BF16 on 1 Gaudi2 card
```

Comment:
I think we should keep the former title as Llava 1.6 is Llava next and most people know it as Llava next.

Reply:
In that case, maybe you can do:
```
### Llava-next

#### llava-v1.6-mistral-7b-hf BF16 on 1 Gaudi2 card

...

#### llava-v1.6-mistral-7b-hf FP8 on 1 Gaudi2 card
```

Reply:
Done